app/assets/src/components/views/bulk_download/BulkDownloadList.jsx

```diff
@@ -33,11 +35,47 @@ const TABLE_COLUMNS = [
   {
     dataKey: "status",
     label: "",
-    width: 240,
+    width: 190,
     cellRenderer: BulkDownloadTableRenderers.renderStatus,
   },
 ];
 
+const STATUS_TYPES = {
+  waiting: null,
```

Comment:
are these supposed to be `null`?

Reply:
Yes, they are supposed to be the default (which renders as a gray label). I can remove them though, so it'll just resolve to undefined.